index.html

```diff
@@ -19,12 +19,33 @@
     <section class="section-box section-box--products">
       <h2 class="section-box__title">Products</h2>
 
+      <div class="section-products__item">
+        <legend class="section-products__item--title">Apple</legend>
+        <small class="section-products__item--description">52kcal per 100g</small>
+      </div>
+
+      <div class="section-products__item">
+        <legend class="section-products__item--title">Chicken and rice</legend>
+        <small class="section-products__item--description">191kcal per 100g</small>
+      </div>
+
+      <div class="section-products__item">
+        <legend class="section-products__item--title">Egg</legend>
+        <small class="section-products__item--description">140kcal per 100g</small>
+      </div>
+      <div class="section-products__item">
+        <legend class="section-products__item--title">Banana joghurt</legend>
+        <small class="section-products__item--description">86kcal per 100g</small>
+      </div>
+
+
     </section>
 
     <section class="main-container">
-      <section class="section-box section-box--add-product">
+      <div class="section-box section-box--add-product">
```

Comment:
Ta zmiana jest celowo czy to jakaś pozostałość z poprzednich wersji?

Reply:
pozostałość, już zmieniam 👍